test/helpers/edge.spec.ts

```diff
@@ -46,12 +46,6 @@ describe('generateRscDataEdgeManifest', () => {
         name: 'RSC data routing',
         path: '/',
       },
-      {
-        function: 'rsc-data',
-        generator: '@netlify/next-runtime@1.0.0',
-        name: 'RSC data routing',
-        path: '/index.rsc',
-      },
```

Comment:
Is this removed due to the cache control? Just checking it needs to be deleted for both the feature flag on and off.

Reply:
More context on those in https://github.com/netlify/next-runtime/pull/2290 which has RSC changes from this PR extracted to standalone change